crypto/x509/x509_vfy.c

```diff
@@ -2430,15 +2430,11 @@ int X509_STORE_CTX_init(X509_STORE_CTX *ctx, X509_STORE *store, X509 *x509,
         ret = X509_VERIFY_PARAM_inherit(ctx->param, store->param);
     else
         ctx->param->inh_flags |= X509_VP_FLAG_DEFAULT | X509_VP_FLAG_ONCE;
+    if (ret == 0)
```

Comment:
get rid of the ret variable if it is just used here and then thrown away...
i.e.
````
if () {
   if (X509-VERIFY...()== 0)
     goto err;
 } else {
   ctx->param->,,,,,, |= ...;
 }
}
````

Reply:
Good spotting.
Done (though this is 'just' refactoring), in a simplified form.